plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/SnapshotsTable.java

```diff
@@ -50,12 +68,12 @@ public SnapshotsTable(SchemaTableName tableName, TypeManager typeManager, Table
         this.icebergTable = requireNonNull(icebergTable, "icebergTable is null");
         tableMetadata = new ConnectorTableMetadata(requireNonNull(tableName, "tableName is null"),
                 ImmutableList.<ColumnMetadata>builder()
-                        .add(new ColumnMetadata("committed_at", TIMESTAMP_TZ_MILLIS))
-                        .add(new ColumnMetadata("snapshot_id", BIGINT))
-                        .add(new ColumnMetadata("parent_id", BIGINT))
-                        .add(new ColumnMetadata("operation", VARCHAR))
-                        .add(new ColumnMetadata("manifest_list", VARCHAR))
-                        .add(new ColumnMetadata("summary", typeManager.getType(TypeSignature.mapType(VARCHAR.getTypeSignature(), VARCHAR.getTypeSignature()))))
+                        .add(new ColumnMetadata(COMMITTED_AT_COLUMN_NAME, TIMESTAMP_TZ_MILLIS))
+                        .add(new ColumnMetadata(SNAPSHOT_ID_COLUMN_NAME, BIGINT))
+                        .add(new ColumnMetadata(PARENT_ID_COLUMN_NAME, BIGINT))
+                        .add(new ColumnMetadata(OPERATION_COLUMN_NAME, VARCHAR))
+                        .add(new ColumnMetadata(MANIFEST_LIST_COLUMN_NAME, VARCHAR))
+                        .add(new ColumnMetadata(SUMMARY_COLUMN_NAME, typeManager.getType(TypeSignature.mapType(VARCHAR.getTypeSignature(), VARCHAR.getTypeSignature()))))
```

Comment:
We can do it as follow up but if the Iceberg snapshot schema adds a column, it'd be nice if we could have it show up automatically here rather than having to add it manually

Reply:
thanks @alexjo2144 for your time to review!
sure, I can work on it as a follow up.